packages/mobile_app/lib/components/userinfo_field.dart

```diff
@@ -0,0 +1,71 @@
+import 'package:flutter/material.dart';
+import 'package:mutable_wrappers/mutable_wrappers.dart';
+import 'package:app/theme/app_colors.dart';
+
+class UserInfoField extends StatefulWidget {
```

Comment:
Might be potentially problematic to maintain our own implementation of user fields, especially if it really not so much different from the default `TextFormField`.
This is not a blocker here, but what we should discuss

Reply:
It saves code size and make it cleaner in screen widgets.
Make your own widgets is a normal practice, yes we can discuss it.

Reply:
It will be not DRY